tests/ecmp/test_fgnhg.py

```diff
@@ -330,6 +341,11 @@ def fg_ecmp(ptfhost, duthost, router_mac, net_ports, port_list, ip_to_port, bank
     logger.info("Send the same flows again, but unshut " + dut_if_shutdown + " and check "
                 "if flows reblanced as expected and are seen on now brought up link")
 
+    #add static neighbor
+    for nexthop, port in list(ip_to_port.items()):
+        if port == shutdown_link:
+            static_neighbor_entry(duthost, nexthop, ptf_to_dut_mac_map[port], prefix_list)
```

Comment:
Shouldn't this happen after you startup the link?

Maybe better to have a link startup function where you do both: startup the link and setup the static neighbor entry in the same function